tools/archive_metadata_validator/docs/archived-study-metadata-handbook.md

```diff
@@ -0,0 +1,303 @@
+
+<pre>
+addition_contact_info: "Tel: +1 555 201-4444"
+</pre>
+
+####Example for concept 2:
+An example for the second concept, i.e. indentation of contained elements, is given here:
+
+<pre>
+study:
+  name: Navigational maps in rat brains
+  dataset:
+    name: fMRI rat navigation
+    url: http://www.example.com/studies/images/rat-navigation
+  purpose: determine how rats learn to navigate in new environments
+</pre>
+
+The code-snippet above defines an element called "study" that contains three sub-elements, i.e. "name", "dataset", and "purpose". Of which "dataset" itself contains two sub-elements, i.e. "name", and "url".
+
+**Please note**: only use spaces for indentation, not tabulators!
+
+####Example for concept 3:
+An example for the third concept, i.e. lists, is given here:
+
+<pre>
+keywords:
+  - fMRI
+  - Rodents
+</pre>
+
+The code snippet above defines an element named "keywords", that contains two text strings, i.e. "fMRI" and "Rodents". Each list element is introduced with a "-", i.e. a minus-sign. 
+
+The content of list elements is not restricted to simple types like text strings or number. List elements themselves can be elements with sub-elements, as shown here:
+<pre>
+study:
+  persons:
+    - person:
+        first_name: John
+        last_name: Harris
+    - person:
+        first_name: Ida
+        last_name: Miller
+</pre>
+The code snippet above defines an element named "study", that contains an element named "persons", that contains a list with two entries. In this example each list entry is itself an element containing other elements. More specific each list entry is a "person" element, that contains two sub-elements, i.e. "first_name" and "last_name" with the respective first and last names.
+
+## Complete Metadata Definition
+Instead of formally describing the MAS format using some kind of schema language, we use the informal description given in the section "Metadata Elements" above and two examples given below. The first example shows a complete metadata description, i.e. a description that contains all elements that MAS defines. The second example is the minimal required set of elements in MAS.
+
+If you keep in mind:
+
+1. All elements marked as optional in section "Metadata Elements" can be left out, no matter whether they contain sub-elements or not.
+
+2. All lists can have arbitrary many entries (at least one), so an arbitrary number of elements can be added, e.g. the persons element could contain a list of 2000 persons.
+
+you should be able to quickly generate a metadata description of your study by modifying the complete example, i.e. modifying element content or deleting elements that you do not need and that are marked as optional above. Please note, the example use four spaces for the next indentation level. Feel free to change this number as long as it is consistent, i.e. sub-elements of an element all have the same indentation.
+
+#### Example 1: Complete Metadata Description
+<pre>
```

Comment:
Did you mean triple backtick? Or is this legit in general (not just in GitHub) for markdown?
Also, tge handbook iirc is rst.

Reply:
I used tripple backtick. It did not work properly on the first line of code (that might be due to the python markdown package). HTML-tags are legit Markdown, so I replace the tripple backticks with <pre>...</pre>, seemed to work then

Reply:
You are right. Existing doc is rst. No problem changing from markdown to rst, imho that is just a simple find/replace, since I do not use very specific markdown commands